src/vivarium_nih_us_cvd/tools/make_artifacts.py

```diff
@@ -49,8 +49,12 @@ def check_for_existing(
     )
     locations = set([sanitize_location(loc) for loc in metadata.LOCATIONS])
```

Comment:
Nit: `locations` is only ever used inside the `location == "all"` block, so you can define it inside.